unskript-ctl/unskript_ctl_run.py

```diff
@@ -752,7 +752,7 @@ def display_action_result(self):
         if self.uglobals.get('info_action_results'):
             for k,v in self.uglobals.get('info_action_results').items():
                 self._banner('')
-                self._highlight(k)
+                print('\033[4m\x1B[1;20;40m' + k + '\x1B[0m\033[0m')
```

Comment:
Why are we not using the utils.TBR_ here as well?

Reply:
TBR_ is a constant and it has the Message already embed in it. I want the Message to be shown in the decorations. Example `k` here is shown with Underline and bold White text. That is why i did not use TBR_ stuff.

Reply:
Done.